docs/guides/storybook-npm-setup.md

```diff
@@ -0,0 +1,155 @@
+
+# Storybook - npm setup - draft
+_Based on PR [#135](https://github.com/bbc/react-transcript-editor/pull/135)_
+
+## webpack 
+Webpack is now used for bundling the component for npm distribution. 
+Biggest change is that the build script now uses webpack instead of simply using babel
+
+```diff
+- "build:component": "rimraf dist && NODE_ENV=production babel  packages/ --out-dir dist --copy-files && rimraf dist/**/*.sample.json dist/**/*.sample.js dist/**/example-usage.js dist/**/*.test.js dist/**/*__tests__ dist/**/*__snapshots__ dist/**/*.spec.js",
++ "build:component": "webpack --config webpack.config.js",
++"pre:publish": "npm run build:component && rm ./dist/package.json || true && cp package.json ./dist/package.json && rm ./dist/README.md || true && cp README.md ./dist/README.md || true ",
++"publish:public": "npm run pre:publish && npm publish dist --access public",
+ ```
```

Comment:
Not entirely sure why this diff is here. Is it so that I can revert back to using babel? Why would I do that?

Reply:
number of reasons, eg storybook proves too slow for dev work, and want to revert back?